5_references/39_server_install.txt

```diff
@@ -0,0 +1,164 @@
+=== Packages organization
+
+==== Packages
+
+Rudder components are distributed as a set of packages.
+
+//image::./images/rudder-packages.png[Rudder packages]
+
+["graphviz", "packages.png"]
+
+.Rudder packages and their dependencies
+
+----
+
+include::../graphviz/packages.dot[]
+
+----
+
++rudder-webapp+::
+
+Package for the Rudder Web Application. It is the graphical interface for
+Rudder.
+
++rudder-inventory-endpoint+::
+
+Package for the inventory reception service. It has no graphical interface. This
+service is using HTTP as transport protocol. It receives an parses the files
+sent by FusionInventory and insert the valuable data into the LDAP database.
+
++rudder-jetty+::
+
+Application server for +rudder-webapp+ and +rudder-inventory-endpoint+. Both
+packages are written in 'Scala'. At compilation time, they are converted into
++.war+ files. They need to be run in an application server. 'Jetty' is this
+application server. It depends on a compatible Java Runtime Environment. It can
+be either 'Oracle Java JRE' or 'OpenJDK 7 JRE'.
+
++rudder-techniquess+::
+
+Package for the Techniques. They are installed in
++/opt/rudder/configuration-repository/techniques+. At runtime, the Techniques are
+copied into a 'git' repository in +/var/rudder+. Therefore, the package depends
+on the +git+ package.
+
++rudder-inventory-ldap+::
+
+Package for the database containing the inventory and configuration information
+for each pending and validated Node. This 'LDAP' database is build upon
+'OpenLDAP' server.  The 'OpenLDAP' engine is contained in the package.
+
++rudder-reports+::
+
+Package for the database containing the logs sent by each Node and the reports
+computed by Rudder. This is a 'PostgreSQL' database using the 'PostgreSQL'
+engine of the distribution. The package has a dependency on the +postgresl+
+package, creates the database named +rudder+ and installs the inialisation
+scripts for that database in +/opt/rudder/etc/postgresql/*.sql+.
+
++rudder-cfengine-community+::
+
+Package for the CFEngine server. This server delivers to the Nodes the Applied
+Policies converted into CFEngine promises.
```

Comment:
This package doesn't exist anymore. It was removed a while back, and now we just have rudder-agent everywhere (on nodes and server). You can just remove this paragraph.